packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -44,26 +44,66 @@ const focusStyle = css`
     )};
 `;
 
-const SwatchList = styled.div.attrs({
-  role: 'listbox',
-})`
-  display: flex;
-  max-width: 100%;
-  flex-wrap: wrap;
-  gap: 6px;
+const Wrapper = styled.div`
+  display: inline-block;
```

Comment:
If we'd change this to `display: flex`, couldn't we use an actual grid for the `SwatchList` instead and not have to add the additional `SwatchWrapper`? Navigating between the swatches is still broken currently using just arrow keys, probably because of adding the extra `SwatchWrapper`.

@barklund Pinging you here for thoughts as well since you implemented the original layout of the color swatches.
Some context for the changes here:
`ColorAdd` (button for adding a new color) was added originally to the `listbox` but since it was a `button` and not `option`, then it was incorrect for accessibility reasons. Sam is now trying to fix it by moving this one button out of the `SwatchList` but still making it group together with other swatches.

Reply:
@miina as far as I can tell the swatch navigation is broken in staging too - I don't think it has to do with this update?

Reply:
@samwhale You're right! Looks like adding the `Tooltip` wrapper around the `StyledSwatch` broke it.